src/redux-store/selectors.js

```diff
@@ -1,3 +1,4 @@
 export const getIsLoggedIn = state => state.auth.isLoggedIn;
 export const getUserRefresh = state => state.auth.isRefresh;
+export const getToken = state => state.auth.token;
```

Comment:
Redux store is alive until user refresh the application(page) unless we're using redux persist to sync the store with browser local storage. If we'll remove redux-persist it stop working. Just a reminder.